Makefile

```diff
@@ -68,6 +69,12 @@ modules: FORCE
 runtime: FORCE
 	cd runtime && $(MAKE)
 
+llvm-runtime-maybe: FORCE
+	-@if [ "llvm" = `./util/chplenv/chpl_llvm.py` ]; then \
```

Comment:
I think you want '$(CHPL_MAKE_HOME)/util/chplenv.chpl_llvm.py' here, rather than just './...'. It seems like everywhere else we run one of the util/chplenv scripts we do this.